src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = century
+    ? people.filter((p) => p.sex === 'm' && Math.ceil(p.died / 100) === century)
+    : people.filter((p) => p.sex === 'm');
```

Comment:
"Avoid single letter names. Be descriptive with your naming." Change the rest of your code to align with this guideline. ([JS Style Guide - 23.1](https://mate-academy.github.io/style-guides/javascript.html#231avoid-single-letter-names-be-descriptive-with-your-naming))
```suggestion
  const men = century
    ? people.filter((person) => person.sex === 'm' && Math.ceil(person.died / 100) === century)
    : people.filter((person) => person.sex === 'm');
```